public/rover.js

```diff
@@ -0,0 +1,58 @@
+const pathOfMarsRover = document.getElementById("path-of-mars-rover");
+const marsRoverSprite = document.getElementById("mars-rover-sprite");
+
+pathOfMarsRover.height = window.innerHeight;
+pathOfMarsRover.width = window.innerWidth;
+
+marsRoverSprite.style.position = "absolute";
+marsRoverSprite.style.left = "60%";
+
+window.addEventListener("resize", (event) => {
+  pathOfMarsRover.height = event.target.innerHeight;
+  pathOfMarsRover.width = event.target.innerWidth;
+});
+
+marsRoverSprite.addEventListener("mousedown", startDraggingImage);
+marsRoverSprite.addEventListener("mousemove", dragTheImage);
+marsRoverSprite.addEventListener("mouseup", stopDraggingImage);
+
+let currentX;
+let currentY;
```

Comment:
One thing I like to do when we have such connected values is to group them into an object

```
const currentPosition = {
    x: 0,
    y: 0
};
```